DevEdu.API/DevEdu.Business.Tests/Data/LessonData.cs

```diff
@@ -0,0 +1,56 @@
+﻿using DevEdu.DAL.Models;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+
+namespace DevEdu.Business.Tests
+{
+    public static class LessonData
+    {
+        public const int ExpectedStudentLeesontId = 42;
+        public static int LessonId = 30;
+        public static int UserId = 42;
+
+        public static StudentLessonDto GetStudentLessonDto()
```

Comment:
please change this to a method that will return an object instead of a list
let's call it "GetSampleStudentLessonDto"